### PR TITLE
[Mikhail Ilichev] Mock 2

### PR DESCRIPTION
**Pull request by miffn3**
Demo here:
https://mikhail-ilichev-mock2.firebaseapp.com/

### CODE REVIEW

File: mock2/private/style/scss/article/article.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  &__image_introduction {
+    max-height: 509px;
+    margin-bottom: 99px;
+  }
+
+  &__label {
+    display: block;
+    margin-bottom: 28px;
+    color: $labelTextColor;
```
Comment (KalachevDev) [inline mock2/private/style/scss/article/article.scss]: I'd suggest you to name color variables like: `color-primary` or `color-secondary`

File: mock2/private/style/scss/colors.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$labelTextColor: #b4ad9e;
+$mainTextColor: #626262;
+$navigationLogo-FooterColor: #000000;
```
Comment (KalachevDev) [inline mock2/private/style/scss/colors.scss]: You need to think about these names

File: mock2/private/style/scss/container/container.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    display: flex;
+    justify-content: center;
+  }
+
+  &_footer {
+    padding: 0 $wrapperPadding;
+  }
+}
+ 
+@media screen and (max-width: 1024px) {
```
Comment (KalachevDev) [inline mock2/private/style/scss/container/container.scss]: Think about creating mixins for media "queries"

File: mock2/private/style/scss/main-content/main-content.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.main-content {
+  min-height: calc(100vh - 196px);
```
Comment (KalachevDev) [inline mock2/private/style/scss/main-content/main-content.scss]: Do you really need to use calc here?

File: mock2/private/style/scss/main.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+$wrapperPadding: 68px;
+
+body {
```
Comment (KalachevDev) [inline mock2/private/style/scss/main.scss]: read about reset.css

File: mock2/public/index.html [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+                        <a href="#" class="article__label">
+                            Leave a comment
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="wrapper">
+            <div class="container container_articles">
+                <div class="article">
```
Comment (KalachevDev) [inline mock2/public/index.html]: Really? Div class article?

File: mock2/public/index.html [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+        <section class="wrapper">
+            <div class="container">
+                <div class="article article_introduction">
+                    <img src="img/article/article_introduction/image.png" alt="Photodiary"
+                        class="article__image article__image_introduction">
+                    <div class="article__description">
+                        <div class="article__label">
+                            Photodiary
+                        </div>
+                        <a href="#" class="article__title">
```
Comment (KalachevDev) [inline mock2/public/index.html:52]: Why not real `h` element?

File: mock2/public/index.html [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+                            The perfect weekend getaway
+                        </a href="#">
```
Comment (KalachevDev) [inline mock2/public/index.html]: you don't need to specify attributes on closing tag

File: mock2/public/index.html [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+                        dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi
+                        tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+                    </div>
+                </div>
+            </div>
+            <div class="container container_articles">
+                <div class="article">
+                    <img src="img/article/image3.png" alt="dream" class="article__image">
+                    <div class="article__label">
+                        photodiary
```
Comment (KalachevDev) [inline mock2/public/index.html]: capital letter

File: mock2/public/index.html [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="wrapper sign-up">
+            <div class="sign-up__title">
+                Sign up for our newsletter!
+            </div>
+            <form action="" class="sign-up__sending-form" method="POST">
+                <input type="email" class="sign-up__input" placeholder="Enter a valid email address" required>
```
Comment (KalachevDev) [inline mock2/public/index.html:203]: input tag might be self-closing

File: mock2/private/style/scss/article/_article.scss [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+  &__title {
+    display: block;
+    margin-bottom: 34px;
+    font-size: 30px;
+    color: $color-text-primary;
+    text-decoration: none;
+  }
+
+  &__title:hover {
```
Comment (KalachevDev) [inline mock2/private/style/scss/article/_article.scss]: &__title {
&:hover
}

File: mock2/private/style/scss/sign-up/_sign-up.scss [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  &__icon {
+    cursor: pointer;
+    width: 32px;
+    height: 17px;
+  }
+
+  &__icon:hover {
+    opacity: .5;
```
Comment (KalachevDev) [inline mock2/private/style/scss/sign-up/_sign-up.scss]: It could be nested to &__icon

File: mock2/private/style/scss/global/_colors.scss [hunk 13]
```diff
@@ -0,0 +1,5 @@
+$color-text-primary: #626262;
+$color-text-secondary: #b4ad9e;
+$color-text-tertiary: #000000;
```
Comment (KalachevDev) [inline mock2/private/style/scss/global/_colors.scss]: doubtful :/

File: mock2/private/style/scss/navigation/_navigation.scss [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+    font-size: 14px;
+    font-weight: 400;
+    margin-left: 100px;
+    line-height: 24px;
+    text-decoration: none;
+    color: $color-text-primary;
+  }
+
+  &__item:hover {
+    opacity: 0.5;
```
Comment (KalachevDev) [inline mock2/private/style/scss/navigation/_navigation.scss]: why it is outside of __item selector?

File: mock2/private/style/scss/navigation/_navigation.scss [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  &__item_logo {
+    margin-left: 0;
+    margin-right: auto;
+    color: $color-text-tertiary;
```
Comment (KalachevDev) [inline mock2/private/style/scss/navigation/_navigation.scss]: same as above

File: mock2/private/style/scss/article/_article.scss [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+    text-decoration: none;
+  }
+
+  &__title {
+    display: block;
+    margin-bottom: 34px;
+    font-size: 30px;
+    color: $color-text-primary;
+    text-decoration: none;
+    &:hover{
```
Comment (KalachevDev) [inline mock2/private/style/scss/article/_article.scss]: missing space

File: mock2/private/style/scss/sign-up/_sign-up.scss [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+    font-size: 30px;
+    letter-spacing: 0.75px;
+    text-align: center;
+  }
+
+  &__sending-form {
+    display: flex;
+    margin-top: 80px;
+    max-width: 400px;
+    border-bottom: 1px solid #000000;
```
Comment (KalachevDev) [inline mock2/private/style/scss/sign-up/_sign-up.scss]: existing color?

File: mock2/private/style/scss/contacts/_contacts.scss [hunk 18]
```diff
@@ -0,0 +1,36 @@
+.contacts {
+  display: flex;
+  padding: 40px 0;
+  justify-content: space-between;
+
+  &__text {
+    margin-right: 10px;
+    color: $color-text-logo-primary;
+    font-family: "Playfair Display";
```
Comment (KalachevDev) [inline mock2/private/style/scss/contacts/_contacts.scss]: You may also create some `include` rule for repeating font family + other properties like font-weight combinations. Try to find common things in your markup / templates and extract them to simplify your life :)

File: mock2/private/style/scss/article/_article.scss (hunk 16 above)
Comment (vadeneev) [reply mock2/private/style/scss/article/_article.scss]: nerd

**Review by KalachevDev — DISMISSED** [review]

**Review by KalachevDev — APPROVED** [review]

File: mock2/private/style/scss/article/_article.scss [hunk 19]
```diff
@@ -1,10 +1,8 @@
 .article {
+  @include font-settings("Playfair Display", 14px, 400, null);
```
Comment (KalachevDev) [inline mock2/private/style/scss/article/_article.scss:2]: Not exactly what I meant but let's keep this

**Review by vovcyan — APPROVED** [review]

**Pull request merged**